conans/test/unittests/model/test_conan_file.py

```diff
@@ -0,0 +1,131 @@
+# coding=utf-8
+
+import os
```

Comment:
This is not testing the ``conanfile``, maybe filename is wrong?

Reply:
It gets the same name as the source file where the function being tested is located. The function will be moved to `conans.model.editable_cpp_info` as suggested in other comments, so the unittest filename will change.